### PR TITLE
Fix TrailingCommaRuleTests for Swift < 4.1

### PR DESCRIPTION
**Pull request by jpsim**
Master is broken because of this.

### CODE REVIEW

File: Source/SwiftLintFramework/Rules/TrailingCommaRule.swift [hunk 1]
```diff
@@ -21,8 +21,8 @@ public struct TrailingCommaRule: ASTRule, CorrectableRule, ConfigurationProvider
         "struct Bar {\n let foo = [1: 2, 2: 3↓, ]\n}\n",
         "let foo = [1, 2, 3↓,] + [4, 5, 6↓,]\n",
         "let example = [ 1,\n2↓,\n // 3,\n]",
-        "let foo = [\"אבג\", \"αβγ\", \"🇺🇸\"↓,]\n",
-        "class C {\n #if true\n func f() {\n let foo = [1, 2, 3↓,]\n }\n #endif\n}"
+        "let foo = [\"אבג\", \"αβγ\", \"🇺🇸\"↓,]\n"
```
Comment (freak4pc) [inline Source/SwiftLintFramework/Rules/TrailingCommaRule.swift]: lol, love the hebrew sample

Comment (jpsim) [reply Source/SwiftLintFramework/Rules/TrailingCommaRule.swift]: :unicode:

Comment (freak4pc) [reply Source/SwiftLintFramework/Rules/TrailingCommaRule.swift]: 🦄

Comment (SDGGiesbrecht) [reply Source/SwiftLintFramework/Rules/TrailingCommaRule.swift]: בבקשה, @freak4pc.

Comment (freak4pc) [reply Source/SwiftLintFramework/Rules/TrailingCommaRule.swift]: תודה :)

**Pull request merged**